states/oklahoma/parse_pdf.py

```diff
@@ -0,0 +1,99 @@
+from common import cache_request
+from io import BytesIO
+from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
+from pdfminer.converter import TextConverter
+from pdfminer.layout import LAParams
+from pdfminer.pdfpage import PDFPage
+from io import StringIO
+import os
+from collections import defaultdict
+
+
+def get_text(url):
+  req = cache_request(url, is_binary=True)
+  with BytesIO(req) as fh:
+    output = StringIO()
+    manager = PDFResourceManager()
+    converter = TextConverter(manager, output, laparams=LAParams())
+    interpreter = PDFPageInterpreter(manager, converter)
+
+    for page in PDFPage.get_pages(fh, set()):
+      interpreter.process_page(page)
+    
+    converter.close()
+    text = output.getvalue()
+    output.close()
+    return text
+
+
+def group_rows(lines):
+  # There is no email address, and fax and phone share a line.
+  sections = ['COUNTY', 'PHONE', 'MAILING ADDRESS']
+
+  # Only add data to output if it is between one of the above sections and an empty string.
+  add_data = False
+
+  rows = {}
```

Comment:
For future reference, I would have made these variables not dictionary values -- you never use the dictionary feature.  But this will do for now.  Feel free to submit another PR to make these variables if you want.